packages/core/src/App/Components/Elements/LandscapeBlocker/landscape-blocker.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { isDisabledLandscapeBlockerRoute } from '@deriv/shared';
+import LandscapeBlockerSvg from 'Assets/SvgComponents/settings/landscape-blocker.svg';
+import './landscape-blocker.scss';
+
+const LandscapeBlocker = () => {
+    const location = useLocation();
+    const pathname = location?.pathname;
+    const is_hidden_landscape_blocker = isDisabledLandscapeBlockerRoute(pathname);
+
+    if (is_hidden_landscape_blocker) return null;
+
+    return (
+        <div id='landscape_blocker' className='landscape-blocker'>
```

Comment:
Are you allowed to use component package here in core?
We can use `<Text/>` and remove font style from css

Reply:
@kate-deriv The same as below, not sure that we really need to modify it because it will be deleted really soon when all packages modified to allow tablet view